addons/website/static/tests/tours/edit_link_popover.js

```diff
@@ -185,21 +187,35 @@ wTourUtils.registerWebsitePreviewTour('edit_link_popover', {
         run: function () {}, // it's a check
     },
     {
-        content: "Ensure that a click on the link popover link opens a new window",
+        content: "Ensure that a click on the link popover link opens a new window in edit mode",
         trigger: 'iframe .o_edit_menu_popover a.o_we_url_link[target="_blank"]',
         extra_trigger: 'iframe .o_edit_menu_popover a.o_we_full_url[target="_blank"]',
-        run: function (actions) {
-            this.$anchor[0].addEventListener('click', (ev) => {
-                const originalPreventDefault = ev.preventDefault.bind(ev);
-                ev.preventDefault = () => {
-                    throw new Error(
-                        "The link popover should not be default prevented as to open a new tab"
-                    );
-                };
-                originalPreventDefault(); // We do not want to open a new tab in a tour
-            }, {once: true, capture: true});
-            actions.click();
+        run: async function (actions) {
+            let resolvePromise;
+            patch(browser, 'window_open_action', {
+                open: (url) => {
+                    if (window.location.hostname === url.hostname && url.pathname.startsWith('/@/')) {
+                        let node = document.querySelector('body');
+                        node.classList.add('new_backend_window_opened');
+                        resolvePromise('The link popover will open link in backend');
+                    }
+                }
+            }, { pure: true }) // We do not want to open a link in a tour
+            await new Promise(async (resolve, reject) => {
+                resolvePromise = resolve;
+                await actions.click();
+                reject("The link is not correct or patch failed");
+            })
+            unpatch(browser, 'window_open_action');
```

Comment:
Do we really need to unpatch?

Reply:
IMO Yes!! @rdeodoo :thinking: 

> It's useless to remove it, what is done in the test is removed/deleted after the test, so this is not needed (and not even awaited anyway)

IDK if it's expected behaviour or not but,
The changes made during the test are getting destroyed once we refresh the page, not after the successful test.
( refer to the attachment )

Do try, 

1. Run the test successfully
2. Click on the popover link ( Do not refresh the page ) 
-> The patch will execute.
3. Refresh the page & click on the popover link
-> Works fine.

I tried to cover the case if the tour runs manually then operations performed without refreshing are not getting blocked. 

If it's okay with this behaviour then there is no need to unpatch.
`async` was to handle the `unpatch()`, If we unpatch then it's needed else no means of that too. 

![test_successful](https://github.com/odoo/odoo/assets/125337508/1ffb5b2c-92ba-4540-abf4-fd1bc81608df)

Reply:
We don't care about running the test locally, it's just a nice to have feature for debugging purpose but this should never be considered when writing a test.
The only thing that needs to be considered is the test being launched through the testing suite on chrome headless in a transaction where everyone is rollbacked after the test is done.